src/test/java/test/resourceTest/ResourcesTest.java

```diff
@@ -0,0 +1,133 @@
+        } catch (TwitterException e) {
+            e.printStackTrace();
+        }
+        int expectedLength = 0;
+        int actualLength = 0;
+        String actualMessage = "";
+        if (status != null) {
+            expectedLength = expectedMessage.length();
+            actualLength = 0;
+        } else {
+            expectedLength = expectedMessage.length();
+            actualLength = actualMessage.length();
+        }
+        Assert.assertEquals(expectedLength, actualLength);
+    }
+
+    @Test
+    public void testcase_getTweets() throws TwitterException {
+        when(restConfig.configurationBuilder()).thenReturn(new ConfigurationBuilder());
+        ArrayList<String> str = new ArrayList<String>();
+        str.add("hii");
+        when(tweetPost.getTweets()).thenReturn(Response.ok(str).build());
+        ArrayList<String> arrayList = new ArrayList<String>();
+        arrayList.add("hii");
+        Response expectedTweet = Response.ok(arrayList).build();
+        Response actualTweet = tweetPost.getTweets();
+        Assert.assertEquals(expectedTweet.getStatus(), actualTweet.getStatus());
+        Assert.assertEquals(expectedTweet.getEntity(), actualTweet.getEntity());
+        System.out.println(expectedTweet);
+        System.out.println(actualTweet);
+    }
+
+    @Test
+    public void testcase_nullTweets_get() {
+        when(tweetPost.getTweets()).thenReturn(Response.ok().build());
+        when(restConfig.configurationBuilder()).thenReturn(new ConfigurationBuilder());
+        Response expectedTweet = Response.ok().build();
+        Response actualTweet = tweetPost.getTweets();
+        Assert.assertEquals(expectedTweet.getEntity(), actualTweet.getEntity());
+        Assert.assertEquals(expectedTweet.getStatus(), actualTweet.getStatus());
+    }
+
+    @Test
+    public void testcase_getTweetstimeline() throws TwitterException {
+        when(restConfig.configurationBuilder()).thenReturn(new ConfigurationBuilder());
+        Twitter twitter = TwitterFactory.getSingleton();
+        ArrayList<String> arrayList = new ArrayList<String>();
+        List<Status> status = twitter.getHomeTimeline();
+        for (Status st : status) {
+            arrayList.add(st.getText());
+        }
+        ArrayList<String> expected = arrayList;
+        ArrayList<String> actual = Controller.timeline_Tweets();
+        Assert.assertEquals(expected, actual);
+    }
+
+    @Test
+    public void testcase_searchTweets() throws TwitterException {
+        when(restConfig.configurationBuilder()).thenReturn(new ConfigurationBuilder());
+        Twitter twitter = TwitterFactory.getSingleton();
+        ArrayList<String> arrayList = new ArrayList<String>();
+        Query query = new Query("source:twitter4j SupriyaChowdar9");
+        QueryResult result = twitter.search(query);
+        for (Status status : result.getTweets()) {
+            System.out.println("@" + status.getUser().getScreenName() + ":" + status.getText());
```

Comment:
Please assert here

Reply:
iam still working on it its bit high end to understand this and add assert to it